lib/private/request.js

```diff
@@ -27,6 +27,10 @@ module.exports = function request(payload) {
     request.set('Cookie', this.cookieHeader)
   }
 
+  if (payload.contentType) {
```

Comment:
Sure, I'll add a comment line there. This endpoint, `/api/admin/permission/rule`, requires `application/json` of Content-Type. So I made it configurable outside of the box. I didn't want to make it hardcoded, `application/json`. Now, it offers us flexibility.

When we don't pass that `contentType`, it acts as before. There shouldn't be any wrong behaviour after this.